### PR TITLE
feat: Add enableTracing Option

### PR DESCRIPTION
**Pull request by HazAT**
Fixes https://github.com/getsentry/sentry-php/issues/1457

### CODE REVIEW

File: src/Options.php [hunk 1]
```diff
@@ -143,6 +149,27 @@ public function getTracesSampleRate(): ?float
...
+
+        $this->options = $this->resolver->resolve($options);
+    }
+
+    /**
+     * Gets if tracing is enabled or not.
+     */
+    public function getEnableTracing(): ?bool
+    {
+        return $this->options['enable_tracing'];
```
Comment (cleptric) [inline src/Options.php:170]: You can run `vendor/bin/phpstan -b` to update the phpstan baseline file.

File: src/Options.php [hunk 2]
```diff
@@ -143,6 +149,27 @@ public function getTracesSampleRate(): ?float
         return $this->options['traces_sample_rate'];
     }
 
+    /**
+     * Sets if tracing should be enabled or not. If null tracesSampleRate takes
+     * precedence.
+     *
+     * @param ?bool $enableTracing Boolean if tracing should be enabled or not
+     */
+    public function setEnableTracing(?bool $enableTracing): void
```
Comment (cleptric) [inline src/Options.php:158]: We should also test this method inside `OptionsTest::testGettersAndSetters`

**Review by cleptric — APPROVED** [review]
Thanks @HazAT :shipit: 🚀

File: src/Options.php (hunk 1 above)
Comment (HazAT) [reply src/Options.php:170]: Yeah, maybe true - it is a bit confusing.
The thing is the logic behind how `isTracingEnabled` behaves works a bit differently than just a simple boolean.
This getter however really returns the plain content.

**Pull request merged**